Fix func_wrapper.test_handle_mixed_function

|   |   |
|---|---|
|tensorflow|<a href="https://github.com/unifyai/ivy/actions/runs/5171292465/jobs/9314744428" rel="noopener noreferrer" target="_blank"><img src=https://img.shields.io/badge/-success-success></a>
|torch|<a href="https://github.com/unifyai/ivy/actions/runs/5171292465/jobs/9314744428" rel="noopener noreferrer" target="_blank"><img src=https://img.shields.io/badge/-failure-red></a>
|numpy|<a href="https://github.com/unifyai/ivy/actions/runs/5171292465/jobs/9314744428" rel="noopener noreferrer" target="_blank"><img src=https://img.shields.io/badge/-success-success></a>
|jax|<a href="https://github.com/unifyai/ivy/actions/runs/5171292465/jobs/9314744428" rel="noopener noreferrer" target="_blank"><img src=https://img.shields.io/badge/-success-success></a>
|paddle|<a href="https://github.com/unifyai/ivy/actions/runs/5171292465/jobs/9314744428" rel="noopener noreferrer" target="_blank"><img src=https://img.shields.io/badge/-success-success></a>


<details>
<summary>FAILED ivy_tests/test_ivy/test_misc/test_func_wrapper.py::test_handle_mixed_function[cpu-ivy.functional.backends.torch-False-False-x0-weight0-True]</summary>

2023-06-04T22:08:32.1690837Z E   AttributeError: 'list' object has no attribute 'ndim'2023-06-04T22:08:32.1712946Z E   AttributeError: 'list' object has no attribute 'ndim'
</details>

--- a/ivy_tests/test_ivy/test_misc/test_func_wrapper.py
+++ b/ivy_tests/test_ivy/test_misc/test_func_wrapper.py
@@ -115,7 +115,7 @@ def test_handle_mixed_function(x, weight, expected):
         test_fn = "ivy.matmul"
         expected = True
     with patch(test_fn) as test_mock_function:
-        ivy.linear(x, weight)
+        ivy.linear(ivy.array(x), ivy.array(weight))
         assert test_mock_function.called == expected
 
 

